config/findbugs-exclude.xml

```diff
@@ -15,4 +15,11 @@
             <Bug pattern="BC_UNCONFIRMED_CAST_OF_RETURN_VALUE"/>
         </Or>
     </Match>
+    <Match>
+        <Or>
+            <Class name="~.*\.GsonAdapters.*"/>
+            <Class name="~.*\.Immutable.*"/>
+            <Class name="~.*\.Modifiable.*"/>
```

Comment:
My fault. I added for ignoring during my test locally before.

I just have a check that `Modifiable*` class does have a bug pattern `UWF_FIELD_NOT_INITIALIZED_IN_CONSTRUCTOR` and it is a bug of immutables lib. `hashCode` of `Modifiable*` class would raise a NPE if it contains null field.
But it won't affect us since we won't use any of it as map key or set value.

I updated the exclude file.